stdlib/3/tkinter/__init__.pyi

```diff
@@ -2831,6 +2831,12 @@ class OptionMenu(Menubutton):
     # configure, config, cget are inherited from Menubutton
     # destroy and __getitem__ are overrided, signature does not change
 
+class _Image(Protocol):
+    tk: _tkinter.TkappType
+    def __del__(self) -> None: ...
```

Comment:
`__del__` being part of a protocol seems odd, does tk really require it?

Reply:
I...don't really know. If there's documentation anywhere on what an image class should define to be tkinter-compatible, I haven't been able to find it. I just included all the methods that are present on both tkinter.Image and PIL.ImageTk.PhotoImage (except `__str__`).